spec/web_server

```diff
@@ -32,6 +32,7 @@ server.get('/support/browser_file_list.js', function(req, res) {
 server.use('/node_modules', express.static(__dirname + '/../node_modules'));
 server.use('/spec', express.static(__dirname));
 server.use('/browser', express.static(__dirname + '/../browser'));
+server.use('/common/lib/util', express.static(__dirname + '/../common/lib/util'));
```

Comment:
Could this not be `server.use('/common', express.static(__dirname + '/../common'));` so that any common file can be included more easily in future?